src/components/AboutUs/AboutUsOverlay.tsx

```diff
@@ -0,0 +1,76 @@
+import { Container, Fade, SxProps, Theme } from "@mui/material";
+import { create } from "zustand";
+import { devtools } from "zustand/middleware";
+import AboutUs from "./AboutUs";
+import PopupHeader from "../Header/PopupHeader";
+
+interface AboutUsStore {
+  isOpen: boolean;
+  toggle: (_: boolean) => void;
+}
+
+export const useAboutView = create<AboutUsStore>()(
+  devtools(
+    (set) => ({
+      isOpen: false,
+      toggle: (checked: boolean) =>
+        set(() => ({ isOpen: checked }), undefined, { type: "toggle" }),
+    }),
+    { name: "AboutUsStore" }
+  )
+);
+
+export default function AboutUsOverlay() {
+  const aboutView = useAboutView();
+  if (!aboutView.isOpen) return null;
+  return (
+    <Fade in={aboutView.isOpen}>
+      <Container sx={styles.container}>
+        <PopupHeader
+          onBack={() => {
+            aboutView.toggle(!aboutView.isOpen);
+          }}
+        ></PopupHeader>
```

Comment:
Let's change this to a self closing tag.

```suggestion
        />
```